NinchatSDKSwift/Implementations/Utilities/QuestionnaireElementConnector.swift

```diff
@@ -130,7 +131,7 @@ extension QuestionnaireElementConnectorImpl {
             if block.target == "_complete", performClosures {
                 self.onCompleteTargetReached?(block, nil, autoApply); return (nil, -1)
             }
-            if block.target == "_audienceRegisteredTarget", performClosures {
+            if (block.target == "_audienceRegisteredTarget" || block.target == "_close"), performClosures {
```

Comment:
is _audienceRegisteredTarget some sort of internal target?

Reply:
Yes, it is an internal target